hudi-utilities/src/test/java/org/apache/hudi/utilities/functional/TestHoodieDeltaStreamer.java

```diff
@@ -1622,6 +1652,12 @@ public void testParquetDFSSourceWithSchemaFilesAndTransformer() throws Exception
     testParquetDFSSource(true, Collections.singletonList(TripsWithDistanceTransformer.class.getName()));
   }
 
+  @ParameterizedTest
+  @MethodSource("testArguments")
```

Comment:
these 2 should be annotated on `testORCDFSSource()` method itself then this method not needed